src/query/exceptions.hpp

```diff
@@ -211,6 +211,26 @@ class IsolationLevelModificationInMulticommandTxException : public QueryExceptio
       : QueryException("Isolation level cannot be modified in multicommand transactions.") {}
 };
 
+class IsolationLevelModificationInAnalyticsException : public QueryException {
+ public:
+  IsolationLevelModificationInAnalyticsException()
+      : QueryException("Isolation level cannot be modified when storage mode is set to IN_MEMORY_ANALYTICAL.") {}
```

Comment:
Append: "... set to IN_MEMORY_ANALYTICAL. IN_MEMORY_ANALYTICAL mode doesn't provide any isolation guarantees, you can think about it as an equivalent to READ_UNCOMMITED."